tests/test_cmdline.py

```diff
@@ -0,0 +1,62 @@
+import zipline.__main__ as main
+import zipline
+from zipline.testing import ZiplineTestCase
+from zipline.testing.predicates import (
+    assert_equal,
+    assert_raises_str,
+)
+from click.testing import CliRunner
+from zipline.extensions import ExtensionArgs
+
+
+class CmdLineTestCase(ZiplineTestCase):
+
+    def init_instance_fixtures(self):
+        super(CmdLineTestCase, self).init_instance_fixtures()
+
+        # make sure this starts empty
+        # TODO: decide how this needs to be reset
+        zipline.extension_args = ExtensionArgs([])
+
+    def test_parse_args(self):
+
+        e = ExtensionArgs(['arg1=test1', 'arg2=test2'])
+        assert_equal(e.extension_args,
+                     {'arg2': 'test2', 'arg1': 'test1'})
+        assert_equal(e.arg1, 'test1')
+        assert_equal(e.arg2, 'test2')
+
+        msg = 'invalid extension argument 1=test3, ' \
```

Comment:
don't use `\` to continue lines, you can wrap the expression in parentheses like:
```python
msg = (
    'first part of the string'
    ' second part of the string'
)
```